docs/builder_image.md

```diff
@@ -64,6 +64,11 @@ using `--scripts-url` flag or the `io.openshift.s2i.scripts-url` with the format
 `image:///path/in/image`), then the `--destination` flag or the `io.openshift.s2i.destination`
 label applies only to sources and artifacts.
 
+**NOTE2**: In the case you need to assemble the Image using a different user than the
```

Comment:
This should not be a note, it should be formatted similar to lines above, and maybe located in the assemble section?